scripts/appleseedMaya/renderGlobals.py

```diff
@@ -427,12 +432,14 @@ def create(self):
                                 enumeratedItem=self._getAttributeMenuItems("pixelFilter")),
                             attrName="pixelFilter")
                         self._addControl(
-                            ui=pm.floatFieldGrp(
-                                label="Pixel Filter Size", numberOfFields=1),
+                            ui=pm.floatSliderGrp(
+                                label="Pixel Filter Size", field=True, value=1.5, sliderStep=0.5, precision=1, cw=(3,160), minValue=0.5, 
+                                fieldMinValue=0.5, maxValue=6.0, fieldMaxValue=20.0),
```

Comment:
Soft max value is 4.0 in appleseed.studio.

Reply:
ok